src/commands/data/connectors/destroy.ts

```diff
@@ -11,23 +13,34 @@ export default class ConnectorsDestroy extends BaseCommand {
     },
   ]
 
+  static flags = {
+    confirm: flags.boolean({
```

Comment:
Should we consider making them type something, like we do for addons? It could be one of the app names for the addons, or even repeating the connector name. I like the idea of following the established patterns, if we can.

Reply:
I _think_ that was an older, pre-oclif, pattern that has been updated. Let's see what @cafreeman has to say about it.